go-controller/pkg/factory/factory_test.go

```diff
@@ -378,6 +378,7 @@ var _ = Describe("Watch Factory Operations", func() {
 
 		const nodeName string = "mynode"
 		type opTest struct {
+			mu      sync.Mutex
```

Comment:
this is the whole stack trace, the race detector complains when the test checks the opTest type variable, because it was modified recently by the handler
https://pastebin.com/NAqaVNtP